lms/templates/oauth2_provider/authorize.html

```diff
@@ -27,6 +27,14 @@ <h1 class="block-center-heading">{% trans "Authorize" %} {{ application.name }}?
                                 <li>{{ scope }}</li>
                             {% endfor %}
                         </ul>
+
+                        <p>{% trans "These permissions will be provisioned to the following organizations:" %}</p>
```

Comment:
How about: "These permissions will be granted for data from your courses associated with the following content providers:"?